### ISSUE TITLE
unexpected carriage return expecting '\'

### ISSUE THREAD

**Issue by VaughanJ** [hadolint/hadolint#438]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior
Lint issues being returned

### Actual behavior
Error message
/dev/stdin:2:13 unexpected carriage return expecting '\', a new line followed by the next instruction, or at least one space

### Steps to reproduce the behavior
On Windows 10
In Powershell
get-content -Raw  "DockerFile" | docker run --rm -i hadolint/hadolint

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:**

```bash
Haskell Dockerfile Linter v1.18.0-0-g76eee5c
```

**Dockerfile** (if relevant)

```Dockerfile
FROM library/busybox:latest
CMD ["date"]
```

**Comment by lorenzo** [comment]
duplicate of #436 

will fix soon

**Comment by lorenzo** [comment]
This was fixed
